ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-presubmits.yaml

```diff
@@ -406,7 +406,7 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )edge-e2e-ai-operator-ztp-capi,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
I don't know. I saw these, but all I ran was `make update` :shrug: 

Maybe something previously didn't update correctly?

Reply:
I could manually remove these also if you think I should

Reply:
for the most part the prow-job configuration matches the ci-operator configuration, but it's not always the case (especially for the ``always_run`` and ``reporter_config`` fields)
it means we have to manually tweak it from time to time, even when just renaming jobs or maybe doing some other very small modifications (maybe it updated file perms? no idea in your case why it did it)

there were a couple of threads in ``#forum-testplatform`` to handle that, but it doesn't seem like they are going to handle it in some special way. so for now we'll have to try and mirror the same configurations as they were before (otherwise it will run a lot more tests by default)

Reply:
https://coreos.slack.com/archives/CBN38N3MW/p1636627767070800

Reply:
Fixed